cmake/coredumper.cmake

```diff
@@ -0,0 +1,14 @@
+MACRO(MYSQL_CHECK_COREDUMPER)
+  # We build coredumper by default
+  IF (NOT DEFINED WITH_COREDUMPER)
+    SET (HAVE_LIBCOREDUMPER 1)
```

Comment:
This will still result in the "else" clause of the below if, building ps without coredumper? I think this should set WITH_COREDUMPER to 1.

Reply:
It will, but the way you suggested make it easier to understand that if you do not set, the default will be WITH_COREDUMPER